app/assets/javascripts/Components/Modals/graders_distribution_modal.jsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import Modal from "react-modal";
+import PropTypes from "prop-types";
+
+export class GraderDistributionModal extends React.Component {
+  static defaultProps = {
+    override: false,
+  };
+
+  constructor(props) {
+    super(props);
+    this.input = React.createRef();
+  }
+
+  componentDidMount() {
+    this.props.graders.forEach(grader => {
+      this.props.setWeighting(grader._id, 1);
+    });
+    Modal.setAppElement("body");
+  }
+
+  onSubmit = event => {
+    event.preventDefault();
+    console.log(this.input); //logs: { current: null }
```

Comment:
@WeissSa I looked into this and you should try accessing `this.input.current` directly. The console log does display `null`, but if you expand the object in the console (or access `current` directly) you'll get a form element. Not sure why the display seems to be inconsistent.

Reply:
When I try logging this.input.current I get null and if I try inputting this.current.value as a FormData object then I get an error (because null cannot have properties). Is it possible I am doing something else wrong? Could it be that I think our version of React might not allow for useRef? I will push my code again if it helps.